tests/Report/JunitReporterTest.php

```diff
@@ -248,17 +246,21 @@ private function assertJunitXmlSchema($xml)
     {
         $xsdPath = __DIR__.'/../../doc/junit-10.xsd';
 
-        libxml_use_internal_errors(true);
-
         static $errorLevels = array(
             LIBXML_ERR_WARNING => 'Warning',
             LIBXML_ERR_ERROR => 'Error',
             LIBXML_ERR_FATAL => 'Fatal Error',
         );
 
+        $internal = libxml_use_internal_errors(true);
+
         $dom = new \DOMDocument();
 
-        $this->assertTrue($dom->loadXML($xml));
+        $load = $dom->loadXML($xml);
+        if (true !== $load) {
+            libxml_use_internal_errors($internal);
+            $this->assertTrue($load, 'XML loading failed.');
```

Comment:
It cannot and that is fine, it will always fail.
This change makes sure the `libxml_use_internal_errors` is set back to the original state as other tests might depend. I choose to do the always fail assert here so we leave the failing message generation etc. to PHPUnit.
(i.e. if I put in `$this->fail(''XML loading failed)` we would loose the details about the result not being `true` but something else).